utils/Addresses.json

```diff
@@ -1,4 +1,124 @@
+        "chainId": 10,
+        "name": "WETH",
+        "isContract": true
+    },
+    {
+        "addr": "0x13e3Ee699D1909E989722E753853AE30b17e08c5",
+        "chainId": 10,
+        "name": "CHAINLINK_ETH_USD",
+        "isContract": true
+    },
+    {
+        "addr": "0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85",
+        "chainId": 10,
+        "name": "USDC",
+        "isContract": true
+    },
+    {
+        "addr": "0x16a9FA2FDa030272Ce99B29CF780dFA30361E0f3",
+        "chainId": 10,
+        "name": "CHAINLINK_USDC_USD",
+        "isContract": true
+    },
+    {
+        "addr": "0x94b008aA00579c1307B0EF2c499aD98a8ce58e58",
+        "chainId": 10,
+        "name": "USDT",
+        "isContract": true
+    },
+    {
+        "addr": "0xECef79E109e997bCA29c1c0897ec9d7b03647F5E",
+        "chainId": 10,
+        "name": "CHAINLINK_USDT_USD",
+        "isContract": true
+    },
+    {
+        "addr": "0x68f180fcCe6836688e9084f035309E29Bf0A2095",
+        "chainId": 10,
+        "name": "WBTC",
+        "isContract": true
+    },
+    {
+        "addr": "0x718A5788b89454aAE3A028AE9c111A29Be6c2a6F",
+        "chainId": 10,
+        "name": "CHAINLINK_WBTC_USD",
+        "isContract": true
+    },
+    {
+        "addr": "0x1F32b1c2345538c0c6f582fCB022739c4A194Ebb",
+        "chainId": 10,
+        "name": "wstETH",
+        "isContract": true
+    },
+    {
+        "addr": "0x524299Ab0987a7c4B3c8022a35669DdcdC715a10",
+        "chainId": 10,
+        "name": "CHAINLINK_WSTETH_ETH",
+        "isContract": true
+    },
+    {
+        "addr": "0xadDb6A0412DE1BA0F936DCaeb8Aaa24578dcF3B2",
+        "chainId": 10,
+        "name": "cbETH",
+        "isContract": true
+    },
+    {
+        "addr": "0x138b809B8472fF09Cd3E075E6EcbB2e42D41d870",
+        "chainId": 10,
+        "name": "CHAINLINK_cbETH_ETH",
+        "isContract": true
+    },
+    {
+        "addr": "0x9Bcef72be871e61ED4fBbc7630889beE758eb81D",
+        "chainId": 10,
+        "name": "rETH",
+        "isContract": true
+    },
+    {
+        "addr": "0xb429DE60943a8e6DeD356dca2F93Cd31201D9ed0",
+        "chainId": 10,
+        "name": "CHAINLINK_RETH_ETH",
+        "isContract": true
+    },
+    {
+        "addr": "0x4200000000000000000000000000000000000042",
+        "chainId": 10,
+        "name": "OPTIMISM",
+        "isContract": true
+    },
+    {
+        "addr": "0x0D276FC14719f9292D5C1eA2198673d1f4269246",
+        "chainId": 10,
+        "name": "CHAINLINK_OPTIMISM_USD",
+        "isContract": true
+    },
+    {
+        "addr": "0x3c8B650257cFb5f272f799F5e2b4e65093a11a05",
+        "chainId": 10,
+        "name": "VELODROME",
+        "isContract": true
+    },
+    {
+        "addr": "0x0f2Ed59657e391746C1a097BDa98F2aBb94b1120",
+        "chainId": 10,
+        "name": "CHAINLINK_VELODROME_USD",
+        "isContract": true
+    },
+    {
+        "addr": "0x6fBe2692d6E63F453A167cBA89a3C402C491F63a",
```

Comment:
@ElliotFriedman this address is incorrect. It should be [0x355F7b5edbfbfb5cCc7a3C67Dab2f99A72FDDa09](https://app.safe.global/home?safe=oeth:0x355F7b5edbfbfb5cCc7a3C67Dab2f99A72FDDa09)